index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+	
+	<head>
+		<title>PORTFOLIO</title>
+			<link rel="stylesheet" type="text/css" href="portfolio.css">
+			<link rel="stylesheet" type="text/css" href="normalize.css">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body>
+	<nav class="sticky">
+	 	<nav class="navholder">
+			<!--<div class="logo"><a href="#"><img class="logoimg" alt="" src=></a></div>-->
+			<div class="contact"><a href="#contactme">CONTACT ME</a></div>
+			<div class="portfolio"><a href="#portfolio">PORTFOLIO</a></div>
+			<div class="about"><a href="#aboutme">ABOUT ME</a></div>
+			<div class="home"><a href="#">HOME</a></div>
+		</nav>
+		</nav>
+		<div class="introbox">
+			<div>
+			<h1>Hello There</h1>
+			<h3>Welcome to my world!</h3>
+			<p>A short description of my world as a nation can be given here. It's a world with only slightly higher tech, due to the general resources scarcity of the place. There are many land-based colonies, like a few settlements in the southeastern half. Like most. Every single one of these is too small to be considered a nation, but they are still human sized. And they're all just child to me.The geography is mountainous, but there's also a lot of forests, thanks to the many mines spread out around the country. I have also personally researched the science of forest, and it's the second to best one.And this is the old map, and the one I've gotten myself from</p>
```

Comment:
lines have a max length, you can wrap text in the file and HTML will still render it correctly.